graylog2-web-interface/src/components/lookup-tables/CacheForm.tsx

```diff
@@ -171,15 +175,19 @@ const CacheForm = ({ type, saved, title, create, cache, validate, validationErro
                 <fieldset>
                   <Row>
                     <Col mdOffset={3} sm={12}>
-                      {create ? (
-                        <Button type="submit" bsStyle="success">Create Cache</Button>
-                      ) : (!loadingScopePermissions && scopePermissions?.is_mutable) && (
-                        <Button type="submit"
-                                bsStyle="success"
-                                role="button"
-                                name="update">
-                          Update Cache
-                        </Button>
+                      {create && (
+                        <FormSubmit submitButtonText="Create cache"
+                                    submitLoadingText="Creating cache..."
+                                    isSubmitting={isSubmitting}
+                                    isAsyncSubmit
+                                    onCancel={onCancel} />
+                      )}
+                      {(!create && !loadingScopePermissions && scopePermissions?.is_mutable) && (
+                        <FormSubmit submitButtonText="Update cache"
```

Comment:
we can create some var smth like `updatable = !create && !loadingScopePermissions && scopePermissions?.is_mutable` will be a bit cleaner